projects/lib/src/lib/components/error-screen/error-screen.component.scss

```
@@ -0,0 +1,101 @@
+  font-style: normal;
+}
+@font-face {
+  font-family: '72';
+  src: url('/fundamental-styles/72-Regular-full.woff2')
+  format('woff2');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'SAP-icons';
```

Stylelint (17.14.0)

[error] Expected no quotes around "SAP-icons" (font-family-name-quotes)

(font-family-name-quotes)